### PR TITLE
Final blog post voice assistant contest

### PR DESCRIPTION
**Pull request by jlpouffier**
## Proposed change

Add a blog post about our upcoming Voice Assistant Contest

TODO
- [x] Forum category
- [x] Confirm final live stream date
- [x] Change date !

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: NOT APPLICABLE
- Link to parent pull request in the Brands repository: NOT APPLICABLE
- This PR fixes or closes issue: NOT APPLICABLE

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Voice Assistant Contest - Let's build and win some prizes!"
+description: "We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some Home Assistant Green, some Home Assistant SkyConnect, and a chance to be on a livestream with us to talk about your creation."
+date: 2024-01-16 00:00:01
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
date: 2024-01-17 00:00:01
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+date_formatted: "January 17, 2024"
+author: Jean-Loïc Pouffier
+comments: true
+categories: Assist
+og_image: /images/blog/2024-01-11-voice-assistant-contest/banner.png
+---
+
+<p><img src='/images/blog/2024-01-11-voice-assistant-contest/banner.png' class='no-shadow' /></p>
+
+TL;DR: We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some [Home Assistant Green](https://www.home-assistant.io/green/), [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/), and a chance to be on a livestream with us to talk about your work.
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
TL;DR: We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some [Home Assistant Green](/green/), [Home Assistant SkyConnect](/skyconnect/), and a chance to be on a livestream with us to talk about your work.
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+<lite-youtube videoid="99lGuB4J-4o" videotitle="Voice Assistant Contest"></lite-youtube>
+
+2023 was the [Year of the Voice](https://www.home-assistant.io/blog/2022/12/20/year-of-voice/). It was a yearly goal to **let users control Home Assistant in their own language**.
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
2023 was the [Year of the Voice](/blog/2022/12/20/year-of-voice/). It was a yearly goal to **let users control Home Assistant in their own language**.
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+We built some amazing things throughout the year, from a very powerful intent recognizer specifically optimized to run on a small computer like a Raspberry Pi 4, all the way to custom wake words created by our community.
+
+In fact, we believe that what we built during the Year of the Voice allows almost anyone to build a voice assistant that embodies our Open Home values perfectly: Personalized, Private and Sustainable.
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
In fact, we believe that what we built during the Year of the Voice allows almost anyone to build a voice assistant that embodies our Open Home values perfectly: Personalized, Private, and Sustainable.
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Every component of our voice assistants can be heavily customized to fit your language, your style and your needs, fully local options exist for each of them, and finally you can retrofit voice assistant into anything, even a [1970s Walkie Talkie](https://www.youtube.com/shorts/HqYbhk6vq9E) found in a flea market
+
+This is why, after spending a year building great things, we want to celebrate what our voice assistants can enable with the community: We’re launching a voice assistant contest, with great prizes to win!
+
+## Contest Specifics
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
## Contest specifics
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+The only requirement is for you to post on [this forum category](https://community.home-assistant.io/c/voice-assistant-contest/60), this is how you officially participate in the contest.
+
+Winners will be announced on the **15th of March**.
+
+A closing ceremony will take place on the **10th of April** in the form of a livestream. Winners will be invited!
+
+<p><img src='/images/blog/2024-01-11-voice-assistant-contest/contest_timeline.png' class='no-shadow' /></p>
+
+## Contest Categories
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
## Contest categories
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+### Most creative satellite ideas
+
+We have all the ingredients to unleash creativity: Custom sentences, custom wake words, custom voices. This category is here to showcase your smartest ideas.
+We will be judging the creativity, uniqueness, and coherency of the whole package.
+
+Example: [Piitaya’s](https://github.com/piitaya) moving and talking droid.
+
+<lite-youtube videoid="vQ7Hmeume9g" videotitle="Wake word demonstration on ESPHome-based 3D printed droid in Home Assistant | Year of the Voice"></lite-youtube>
+
+### Best Starting Guides
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
### Best starting guides
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+Because our voice assistant can be heavily customized, starting can be daunting. This category is here to showcase the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
Because our voice assistant can be heavily customized, starting can be daunting. This category showcases the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+We will be judging how easy the guide is to follow, and how usable the final product is.
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
We will judge how easy the guide is to follow and how usable the final product is.
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+### Best voice experiences
+
+This category is here for people who want to participate in the contest but not necessarily build a hardware product. This is all about creating the best experience using a voice assistant on Home Assistant.
+We support custom sentences, custom responses, and custom actions.
+Go crazy, be creative, and show us how to unlock the potential of our voice assistants.
+We will be judging how useful the experience is, how easy it is to integrate it, and of course, of fun it is.
+
+Example: [Play any music on any speaker in your home from a blueprint.](https://blog.jlpouffier.fr/chatgpt-powered-music-search-engine-on-a-local-voice-assistant/)
+
+## Contest Prizes
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
## Contest prizes
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+For each category, the winner will receive:
+
+- a [Home Assistant Green](https://www.home-assistant.io/green/)
+- a [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/)
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
- A [Home Assistant Green](/green/)
- A [Home Assistant SkyConnect](/skyconnect/)
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+- A spot on the closing ceremony livestream to present his/her work, on the 10th of April
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
- A spot on the closing ceremony livestream to present his/her work on the 10th of April
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+
+Runners-up will receive a [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/).
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
Runners-up will receive a [Home Assistant SkyConnect](/skyconnect/).
```

File: source/_posts/2024-01-17-voice-assistant-contest.markdown [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+The community will also get a chance to vote for a favorite entry, the choice of the community will also receive the same prizes as a category winner (A [Home Assistant Green](https://www.home-assistant.io/green/), A [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/), a spot on the livestream on the 10th of April).
```
Comment (frenck) [inline source/_posts/2024-01-17-voice-assistant-contest.markdown]:
```suggestion
The community will also get a chance to vote for a favorite entry. The choice of the community will also receive the same prizes as a category winner (A [Home Assistant Green](/green/), A [Home Assistant SkyConnect](/skyconnect/), a spot on the livestream on the 10th of April).
```

**Review by frenck — CHANGES_REQUESTED** [review]
Some small things spotted.

**Review by frenck — APPROVED** [review]
Looks good, thanks @jlpouffier 👍 

../Frenck
